WatchdogService/CleanupOperations.cs

```diff
@@ -261,7 +262,7 @@ internal async Task CleanupDiagnosticTablesAsync()
         /// </summary>
         /// <param name="table">CloudTable instance.</param>
         /// <returns>Number of items removed.</returns>
-        internal async Task<int> EnumerateTableItemsAsync(CloudTable table)
+        internal async Task<int> EnumerateTableItemsAsync(TableItem table, TableServiceClient client)
```

Comment:
If we accept the TableClient, we have all the context needed.
```suggestion
        internal async Task<int> EnumerateTableItemsAsync(TableClient client)
```